reference/blueprint-api/FindOne.md

```diff
@@ -18,7 +18,7 @@ If the action was triggered via a socket request, the requesting socket will be
  ---------------------------------- | --------------------------------------- |:---------------------------------
  model          | ((string))   | The [identity](https://sailsjs.com/documentation/concepts/models-and-orm/model-settings#?identity) of the containing model.<br/><br/>e.g. `'purchase'` (in `/purchase/7`)
  id                | ((string))    | The desired target record's primary key value<br/><br/>e.g. `'7'` (in `/purchase/7`).
- _populate_       | ((string))    | If specified, overide the default automatic population process. Accepts a comma separated list of attributes names for which to populate record values. See [here](https://sailsjs.com/documentation/concepts/models-and-orm/records#?populated-values) for more information on how the population process fills out attributes in the returned record according to the model's defined associations.
+ _populate_       | ((string))    | If specified, overide the default automatic population process. Accepts a comma separated list of attributes names for which to populate record values, or specify `false` to have no attributes populated. See [here](https://sailsjs.com/documentation/concepts/models-and-orm/records#?populated-values) for more information on how the population process fills out attributes in the returned record according to the model's defined associations.
```

Comment:
same typo as above  (unrelated to this PR really, since it was there before, just might as well nab it)